core/scripts/cli/sponsor/create.js

```diff
@@ -0,0 +1,43 @@
+const inquirer = require("inquirer");
+const getDefaultAccount = require("../wallet/getDefaultAccount");
+
+const create = async (web3, artifacts, emp) => {
+  const ExpandedERC20 = artifacts.require("ExpandedERC20");
+  const { toWei, fromWei, toBN } = web3.utils;
+
+  const sponsorAddress = await getDefaultAccount(web3);
+
+  const scalingFactor = toBN(toWei("1"));
+  // TODO: Understand why we need a .rawValue in one case but not the other.
+  const totalPositionCollateral = toBN((await emp.totalPositionCollateral()).rawValue.toString());
+  const totalTokensOutstanding = toBN((await emp.totalTokensOutstanding()).toString());
+  if (totalTokensOutstanding.isZero()) {
+    console.log("Error: can't create initial position with this tool");
+  }
+  const gcr = totalPositionCollateral.mul(scalingFactor).divRound(totalTokensOutstanding);
+
+  const input = await inquirer.prompt({
+    message: "How many tokens to create?",
+    name: "tokensCreated",
+    validate: value => value > 0 || "Number of tokens must be positive"
```

Comment:
does this implicitly cast `value` to a number for the validate() call?

Reply:
Yeah gotta love js.